portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,26 @@
 <!DOCTYPE html>
 <html>
+
   <head>
     <meta charset="UTF-8">
     <title>My Portfolio</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
   </head>
+
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
+    <section>
+      <nav>
+        <a href="#" class="name">ANGEL WALIA</a>
+        <ul>
+          <li> <a href="#" class="active">Home</a></li>
+          <li> <a href="#">Projects</a></li>
+          <li> <a href="#">About</a></li>
+          <li> <a href="#">Contact</a></li>
+          <li> <a href="#">Resume</a></li>
```

Comment:
nit: "<li> <a" should be "<li><a", any reason for the space between the '>' and '<'?

Reply:
noted!